client/web/src/repo/commits/GitCommitNode.tsx

```diff
@@ -194,8 +195,9 @@ export const GitCommitNode: React.FunctionComponent<React.PropsWithChildren<GitC
                                     className={styles.shaAndParentsCopy}
                                     onClick={() => copyToClipboard(parent.oid)}
                                     data-tooltip={flashCopiedToClipboardMessage ? 'Copied!' : 'Copy full SHA'}
+                                    aria-label={flashCopiedToClipboardMessage ? 'Copied!' : 'Copy full SHA'}
```

Comment:
There's extra work we need to do to make the announcement more accessible to screen readers but we have an open issue for that: https://github.com/sourcegraph/sourcegraph/issues/34973 

Let's keep it as this for now:
```suggestion
                                    aria-label="Copy full SHA"
```